go/kbfs/simplefs/simplefs.go

```diff
@@ -587,42 +586,49 @@ func (k *SimpleFS) refreshSubscription(
 		return nil
 	}
 
+	tlfType, tlfName, _, _, err := remoteTlfAndPath(path)
+	if err != nil {
+		return err
+	}
+	tlfPathFromGUI := stdpath.Join("/keybase", tlfType.PathString(), tlfName)
```

Comment:
You could use `libkbfs.BuildCanonicalPath` for this (or, soon, `tlfhandle.BuildCanonicalPathForTlfName`).

Reply:
This has to be the path that GUI tracks for notifications to work properly, so it might not necessarily be canonical path. I guess it'd still work with the function, just feels weird (confusing?) to pass in a non-canonical name as `tlf.CanonicalName`.

Reply:
Would you rather I use `tlfHandle.BUildCanonicalPath` and pass in the non-canonical TLF name? This one accepts strings rather than tlf.CanonicalName.

Reply:
`tlfhandle.BuildCanonicalPath` is fine with me if you prefer it. The "canonical" here refers to the formatting of `/keybase/type/something`, not that the tlf name itself is canonical, so it seems fine to me.  But feel free to rename it or add to the comments as needed.